pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/HashUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.pnc.bacon.pig.impl.utils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.MessageDigest;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.jboss.resteasy.util.Hex;
+
+public class HashUtils {
+    private HashUtils() {
+    }
+
+    public static String hashDirectory(Path directory) {
+        MessageDigest sha = DigestUtils.getSha512Digest();
+        try {
+            Files.walk(directory).filter(Files::isRegularFile).sorted().forEach(path -> {
```

Comment:
I think this should be `try (FIles.walk()) {`

Reply:
I'll fix it in #402

Reply:
thanks!